assembl/graphql/landing_page.py

```diff
@@ -32,18 +34,24 @@ def resolve_title_entries(self, args, context, info):
 
 
 class LandingPageModule(SecureObjectType, SQLAlchemyObjectType):
+    __doc__ = docs.LandingPageModule.__doc__
 
     class Meta:
         model = models.LandingPageModule
         interfaces = (Node, )
         only_fields = ('id', 'enabled', 'order', 'configuration')
 
+<<<<<<< HEAD
     module_type = graphene.Field(LandingPageModuleType)
     exists_in_database = graphene.Boolean()
     title = graphene.String(lang=graphene.String())
     title_entries = graphene.List(LangStringEntry)
     subtitle = graphene.String(lang=graphene.String())
     subtitle_entries = graphene.List(LangStringEntry)
+=======
+    module_type = graphene.Field(LandingPageModuleType, description=docs.LandingPageModule.module_type)
+    exists_in_database = graphene.Boolean(description=docs.LandingPageModule.exists_in_database)
+>>>>>>> Update landing page docstrings
```

Comment:
You seems to have left a conflict here.

Reply:
Oops, that must have been left from the rebase. Fixing now.